include/glow/Base/Type.h

```diff
@@ -365,7 +368,8 @@ struct Type final {
   /// Notice that we don't consider Int64ITy as an integer because we are not
   /// performing calculations on this type.
   bool isQuantizedType() const {
-    return isType<int8_t>() || isType<int16_t>() || isType<int32_t>();
+    return isType<int8_t>() || isType<int16_t>() ||
```

Comment:
can you update comments above?
can you make these checks consistent, e.g., rely on elementType_ .